client/app/components/Resource/__tests__/Resource.spec.jsx

```diff
@@ -17,68 +17,49 @@ const AUTHOR = 'Desi Rottman';
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(false);
-
-    wrapper = shallow(<Resource title={TITLE} link={URL} author={AUTHOR} />);
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(false);
-
-    wrapper = shallow(
-      <Resource title={TITLE} link={URL} tags={TAGS} author={AUTHOR} />,
-    );
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(false);
+  const { getByRole, getByText } = screen;
+  test('it renders with neither tags nor author', () => {
+    render(<Resource title={TITLE} link={URL} />);
+    expect(getByText(TITLE)).toBeInTheDocument();
+    expect(getByRole('link')).toBeInTheDocument();
   });
-
-  it('renders with only tags', () => {
-    let wrapper = shallow(
-      <Resource title={TITLE} link={URL} tagged tags={TAGS} />,
-    );
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(true);
-    expect(wrapper.find('Tag').length).toEqual(TAGS.length);
-
-    wrapper = shallow(
-      <Resource title={TITLE} link={URL} tagged tags={EMPTY_TAGS} />,
-    );
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(true);
-    expect(wrapper.find('Tag').length).toEqual(0);
+  test('it renders with tags={TAGS} and no tagged prop', () => {
+    render(<Resource title={TITLE} link={URL} tags={TAGS} />);
+    expect(getByText(TITLE)).toBeInTheDocument();
+    expect(getByRole('link')).toBeInTheDocument();
   });
-
-  it('renders with only author', () => {
-    const wrapper = shallow(
-      <Resource title={TITLE} link={URL} external author={AUTHOR} />,
-    );
-    expect(wrapper.find('.author').exists()).toEqual(true);
-    expect(wrapper.find('.tags').exists()).toEqual(false);
+  test('it renders with tagged and tags={TAGS}', () => {
+    render(<Resource title={TITLE} link={URL} tagged tags={TAGS} />);
+    expect(getByText(...TAGS));
+    expect(getByText(TITLE)).toBeInTheDocument();
+    expect(getByRole('link')).toBeInTheDocument();
   });
-
-  it('renders with tags and author', () => {
-    let wrapper = shallow(
-      <Resource title={TITLE} link={URL} external tagged />,
-    );
-    expect(wrapper.find('.author').exists()).toEqual(true);
-    expect(wrapper.find('.author').text()).toEqual('');
-    expect(wrapper.find('.tags').exists()).toEqual(true);
-    expect(wrapper.find('Tag').length).toEqual(0);
-
-    wrapper = shallow(
+  test('it renders with tags={EMPTY_TAGS} and no author ', () => {
+    render(<Resource title={TITLE} link={URL} tagged tags={EMPTY_TAGS} />);
+    expect(getByRole('link')).toBeInTheDocument();
+    expect(getByText(TITLE)).toBeInTheDocument();
+  });
+  test('it renders with external and author={AUTHOR}', () => {
```

Comment:
In a similar tone as above, we don't need `external` and the `author` prop should be enough.